databases/core.py

```diff
@@ -168,6 +172,9 @@ def __init__(self, backend: DatabaseBackend) -> None:
     async def execute_many(self, query: ClauseElement, values: list) -> None:
         await self._connection.execute_many(query, values)
 
+    async def raw_connection(self) -> typing.Any:
+        return await self._connection.raw_connection()
```

Comment:
Let's go with...

```python
@property
def raw_connection(self) -> typing.Any:
    return self._connection.get_raw_connection()
```

Reply:
Also let's move the implementation to the very end of the class. That layout feels more natural to me.

Reply:
Happy with layout, but considering the given context I think I'm looking forward to:

```
@property
def raw_connection(self) -> typing.Any:
    return self._connection.raw_connection
```

if there're no objections

Reply:
That's fine with me, yup. 👍